procedures/candlepin/branch.md.erb

```diff
@@ -0,0 +1,28 @@
+# Branch Packaging
+
+- [ ] Clone https://github.com/theforeman/candlepin-packaging
+- [ ] Check out most recent branch rpm/< latest_version >
+- [ ] Create a new branch for the version being added
+  - `git checkout -b rpm/<%= release %>`
+- [ ] Update `package_manifest.yaml` with <%= release %>
+- [ ] Push new branch to https://github.com/theforeman/candlepin-packaging
+
+# Update CI
+
+- [ ] Open a PR with the result of [jenkins-jobs](https://github.com/theforeman/jenkins-jobs) branching: `./branch-foreman <%= release %> KATELLO_VERSION`
+
+# Build Package
```

Comment:
Is building a package part of the branching? I'd assume that it's separate because it can also happen within a branch.